### DISCUSSION TITLE
pip request talk recording

### DISCUSSION THREAD

**Discussion by Zoey2936** [nextcloud/all-in-one#3307] [Questions] [answered]
do we need this line? https://github.com/nextcloud/spreed/blob/864e3b180fbd868dbfe0bf2a45bd87b90266d654/recording/docker-compose/Dockerfile#L13C1-L13C1

**Comment by szaimen** [comment]
Based on https://github.com/nextcloud/spreed/pull/9177/files#r1178635478 I think yes?

**Reply by Zoey2936** [reply]
not sure, we don't get this error in the logs

**Reply by szaimen** [reply]
Ah did you already try without and there werent any errors?

**Reply by Zoey2936** [reply]
https://github.com/nextcloud/spreed/pull/9177/files#r1178672725 no I never had seen this logs

**Reply by szaimen** [reply]
All right, lets try to remove this then. Can you please create the PR? :)

**Reply by Zoey2936** [reply]
I thought we don't have this line?

**Reply by szaimen** [reply]
Ah right, I was confused now 😅

**Comment by szaimen** [comment, marked as answer]
So seems like we dont need it
